tokio/src/net/windows/named_pipe.rs

```diff
@@ -851,6 +851,48 @@ impl NamedPipeServer {
     ) -> io::Result<R> {
         self.io.registration().try_io(interest, f)
     }
+    
+    /// Reads or writes from the socket using a user-provided IO operation.
```

Comment:
The underlying IO resource here is not a socket, it's a pipe. Please change the docs in this file to use the right term.